terraform-module/modules/frontend-spa-edge-lambda/main.tf

```diff
@@ -11,7 +11,7 @@ resource "local_file" "lambda_config" {
   content = jsonencode({
     "originBucketName"         = var.bucket_name
     "originBucketRegion"       = var.bucket_region
-    "previewDeploymentPostfix" = var.env == "staging" ? ".${var.domain_name}" : ""
+    "previewDeploymentPostfix"  = var.env == "production" ? "" : ".${var.domain_name}"
```

Comment:
Why are we flipping the ternary here? It would be ideal to remove the dynamic value here all together and just supply an explicit var for `previewDeploymentPostfix`. In an effort to make our modules more env agnositc, we typically avoid this partner of calculating a value based on the env. It's made deploying to new envs a nightmare.

Reply:
> It's made deploying to new envs a nightmare.

Yeah I'm experiencing that first hand 😄 

I'm flipping the ternary so that I don't need to change this to `var.env == "staging" || var.env == "product-dev"`.

I believe I'm going to need this change in order to make feature deployments to product-dev work.

I created a follow up ticket to drop the ternary completely
https://linear.app/pleo/issue/WEB-656/stop-using-env-ternary-in-edge-lambda-config-file

Let me know if I should rather remove this change completely from this PR and fix the underlying issue by removing the ternary in that

Reply:
All good, as long as we do it eventually.